lighthouse-core/config/config-helpers.js

```diff
@@ -34,6 +35,14 @@ const mergeOptionsOfItems = function(items) {
   return mergedItems;
 };
 
+/**
+ * @param {unknown} value
+ * @return {value is string|LH.IcuMessage}
+ */
+function isStringOrIcuMessage(value) {
```

Comment:
should this just live on i18n or you feel like it won't come up again?

Reply:
> should this just live on i18n or you feel like it won't come up again?

it seemed like just an internal helper fn for config (since the inline check was too long for the line and repeated 3x), so it's just scoped to the module, but I don't have strong feelings about keeping it locked away.